composer.json

```diff
@@ -4,21 +4,14 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.0",
+        "dpc-sdp/tide_core": "dev-feature/SDPA-1213_metatag_upgrade as 1.0",
         "drupal/token_conditions": "dev-1.x",
-        "drupal/webform": "5.0.0-rc18"
+        "drupal/webform": "^5.2"
```

Comment:
@sonnykt , are we really restricted to ^5.2 in this module, or ^5.0 and we want ^5.2 in content-vic.

Reply:
@kurtfoster 5.2 has major changes in underlying backend code compared to 5.0. It adds proper support for JSONAPI. We need 5.2 to work with JSONAPI 2.x. This version should not have any changes in frontend though.